src/App.test.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
```

Comment:
```suggestion
import React from 'react';

import { useSelector, useDispatch } from 'react-redux';
```

여긴 구분해주는 게 좋을 것 같아요.